pkg/windows/dockerfile

```diff
@@ -32,7 +32,8 @@ COPY pyinstaller-requirements.txt c:/temp/
 COPY hubble.conf C:/temp/
 #install Chocolatey, then git and git.portable, and osquery
 RUN powershell.exe -Command Set-ExecutionPolicy Bypass -Scope Process -Force; iex ((New-Object System.Net.WebClient).DownloadString("$env:CHOCO_URL"))
-RUN powershell.exe -Command choco install git git.portable osquery nssm -y;
+RUN powershell.exe -Command choco install git osquery nssm -y;
+RUN powershell.exe -Command choco install git.portable --version 2.19.0 -y;
```

Comment:
There is some problem with the latest version of portable git.
It is not able to fetch profiles from git.
The older version is working fine.
Anyways, it's a good idea to keep our dependencies pinned to a specific version so that we get identical build every time we run docker.